test/TestImport.hs

```diff
@@ -36,16 +37,20 @@ withApp = before $ do
 -- 'withApp' calls it before each test, creating a clean environment for each
 -- spec to run in.
 wipeDB :: App -> IO ()
-wipeDB app = do
-    runDBWithApp app $ do
-        tables <- getTables
-        sqlBackend <- ask
+wipeDB app = runDBWithApp app $ do
+    tables <- getTables
+    sqlBackend <- ask
 
-        let escapedTables = map (connEscapeName sqlBackend . DBName) tables
-            query = "TRUNCATE TABLE " ++ (intercalate ", " escapedTables)
-        rawExecute query []
+    let escapedTables = map (connEscapeName sqlBackend . DBName) tables
+        query = "TRUNCATE TABLE " ++ intercalate ", " escapedTables
+    rawExecute query []
 
 getTables :: MonadIO m => ReaderT SqlBackend m [Text]
 getTables = do
-    tables <- rawSql "SELECT table_name FROM information_schema.tables WHERE table_schema = 'public';" []
-    return $ map unSingle tables
\ No newline at end of file
+    tables <- rawSql [st|
+        SELECT table_name
+        FROM information_schema.tables
+        WHERE table_schema = 'public';
+    |] []
+
+    return $ map unSingle tables
```

Comment:
Based on running `cabal test` on this branch, `shakespeare` needs to be added to the build-depends of the test project to import `Text.Shakespeare.Text`. 

Also, not huge deal, but it does make the SQL log output a little uglier:

```
14/Mar/2015:14:52:16 -0700 [Debug#SQL] "\n        SELECT table_name\n        FROM information_schema.tables\n        WHERE table_schema = 'public';\n    " [] @(persistent-2.1.1.4:Database.Persist.Sql.Raw ./Database/Persist/Sql/Raw.hs:42:26)
```